cubic-server/protocol/serialization/addPrimaryType.hpp

```diff
@@ -185,6 +185,11 @@ namespace protocol
     {
         addLong(out, ((data.x & 0x3FFFFFF) << 38) | ((data.z & 0x3FFFFFF) << 12) | (data.y & 0xFFF));
     }
+
+    constexpr void addBitSet(std::vector<uint8_t> &out, const std::vector<uint64_t> &bitset)
```

Comment:
Why not just use `addArray<uint64_t, addUnsignedLong>` ?

This can be confusing because we now have the `popBitSet` which return an `std::bitset`

Reply:
That was old code, it isn't getting executed.
Getting rid of it